ceph/rados/rados_scrub.py

```diff
@@ -189,3 +190,17 @@ def set_osd_flags(self, flag, value):
             return True
         log.error(f"Failed to {flag} OSD flag {value}")
         return False
+
+    def get_dump_scrubs(self, osd_id):
+        """
+        Method is used to return the osd dump scrubs
+         Args:
+             osd_id: osd id number
+        Returns:
+              Dump scrub output in the json format.
+        """
+        base_cmd = f"cephadm shell --name osd.{osd_id} ceph daemon  osd.{osd_id}  dump_scrubs -f json 2>/dev/null"
```

Comment:
Do we really need to tap into the OSD container in order to run this command?

Reply:
Yes Harsh.